hunting/macos/queries/suspicious_network_connections_by_unsigned_macho.toml

```diff
@@ -0,0 +1,27 @@
+[hunt]
+author = "Elastic"
+integration = "logs-endpoint.events.network-*"
+uuid = "44aff0e3-e0d7-4dca-a94f-2dd0b96f18bd"
+name = "Suspicious Network Connections by unsigned machO"
+language = "ES|QL"
+license = "Elastic License v2"
+notes = [
+    "This hunt aggregate by process Id and destination ip the number of connections per hour over a period of time greater than a defined threshold. The process paths are scoped to Microsoft signed binaries often injected or used as a lolbin to masquerade malicious execution. This could be a sign of long term network activity to perform command and control from an injected process.", "Hunt can be extended by adding suspicious process paths or lolbins.",
```

Comment:
```suggestion
    "This hunt aggregates by process ID and destination IP the number of connections per hour over a period of time greater than a defined threshold. The process paths are scoped to Microsoft signed binaries often injected or used as a lolbin to masquerade malicious execution. This could be a sign of long term network activity to perform command and control from an injected process.", "Hunt can be extended by adding suspicious process paths or lolbins.",
```